site/content/docs/alerts/90034.md

```diff
@@ -24,8 +24,12 @@ Do not trust any user data in NGINX configs. In this case it is probably the use
 
 * https://www.nginx.com/blog/trust-no-one-perils-of-trusting-user-input/
 
+### CWE: [0](https://cwe.mitre.org/data/definitions/0.html)
+
+### WASC:  0
+
 ### Code
 
  * [org/zaproxy/zap/extension/ascanrulesAlpha/CloudMetadataScanner.java](https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/CloudMetadataScanner.java)
```

Comment:
Looks like some of the changes have been missed.

Was this assembled manually or automatically?

Reply:
Looks like the script wasnt updated correctly .. fixing that now...